### DISCUSSION TITLE
Building openssl for linux-armv4 platform on version 3.3.1, running prompts segmentation fault

### DISCUSSION THREAD

**Discussion by aaron230112** [openssl/openssl#25248] [Q&A]
1，The same configuration cross-compilation builds openssl for the linux-armv4 platform. It runs normally on version 1.0.2j, but on version 3.3.1, after successful compilation, running openssl prompts segmentation fault. 
2，If do not specify the platform type (default is linux-x86_64) in version 3.3.1, run openssl without interactive command line (OpenSSL>), and the help information will appear directly (expecting the interactive command line OpenSSL> to appear)
BTW, the dependent ssl and crypto dynamics are successfully connected

<img width="946" alt="works fine on version 1 0 2j" src="https://github.com/user-attachments/assets/df783f3f-01f2-4d4a-9702-30ecc7232eff">

<img width="356" alt="works error on version 3 3 1" src="https://github.com/user-attachments/assets/5ddb1d8a-a019-4aee-aa24-a3c4470891ec">

[perl-configdata-pm-v3.3.1.txt](https://github.com/user-attachments/files/16689278/perl-configdata-pm-v3.3.1.txt)

**Comment by nhorman** [comment]
can you provide a backtrace?

**Reply by aaron230112** [reply]
> can you provide a backtrace?

Could you tell me what I should do, thank you。

**Reply by nhorman** [reply]
1) configure openssl with the --debug flag
2) run your curl command under gdb, ensuring LD_LIBRARY_PATH points to the newly built library
3) When the segfault is encountered, enter the command "bt" at the gdb prompt and post the results here
